python/tools/BUILD.bazel

```diff
@@ -0,0 +1,11 @@
+load("@semmle_code//:dist.bzl", "pack_zip")
+
+pack_zip(
+    name = "tools",
+    srcs = glob(["**/*"]),
+    excludes = [
+        "BUILD.bazel",
+    ] + glob(["recorded-call-graph-metrics/**"]),
```

Comment:
I believe this is a functional change compared to the previous build. Is this on purpose?

Reply:
If you mean the exclusion of the `recorded-call-graph-metrics` directory, then yes. The reason for this is that this directory already existed as a subdirectory of the `tools` directory, and hence without the exclusion would get included in the extractor distribution (which was silly).

Reply:
Ah, I didn't ready properly - I thought it was an active _inclusion_ of the directory, which is a functional change. Having an active _exclusion_ for it is sensible, to keep the dist as it was previously. Thanks for the clarification!